Source/spells.cpp

```diff
@@ -113,6 +113,16 @@ bool IsWallSpell(spell_id spl)
 	return spl == SPL_FIREWALL || spl == SPL_LIGHTWALL;
 }
 
+bool TargetsMonster(spell_id id)
+{
+	return id == SPL_FIREBALL
+	    || id == SPL_FIREWALL
+	    || id == SPL_FLAME
+	    || id == SPL_LIGHTNING
+	    || id == SPL_STONE
+	    || id == SPL_WAVE;

```

Comment:
This is what took the longest to review, it's so confusing that fire `SPL_FIREBALL` targets monsters, but `SPL_FIREBOLT` does not :D

Reply:
Hmm, that's a good point. I hope this doesn't get used for anything other than scrolls...